File: Go/Palindromo/palindromo.go
package main

import (
	"fmt"
	"strings"
)

func main() {
	var word string
	fmt.Print("Digite uma palavra: ")
	fmt.Scan(&word)

	isPalindrome := true
	word = strings.ToLower(word)

	for i := 0; i < len(word)/2; i++ {
		if word[i] != word[len(word)-i-1] {
			isPalindrome = false
			break
		}
	}

	if isPalindrome {
		fmt.Printf("%s é um palíndromo\n", word)
	} else {
		fmt.Printf("%s não é um palíndromo\n", word)
	}
}

File: Python/Palíndromo/palindromo.py
palavra = input("Digite uma palavra: ")

# remover espaços em branco e transformar em letras minúsculas
palavra = palavra.replace(" ", "").lower()

# verificar se a palavra é um palíndromo
if palavra == palavra[::-1]:
    print("A palavra é um palíndromo.")
else:
    print("A palavra não é um palíndromo.")


